### DISCUSSION TITLE
linux下一直无法加载URL

### DISCUSSION THREAD

**Discussion by bluecodeboy1987** [CefView/QCefView#324] [Q&A]
您好，请教一下，基于89.0版本的，linux平台下采用qt5.5 OSR模式下加载URL后 onPaint函数一直无法触发是什么原因造成的呢，谢谢

**Comment by L-Super** [comment]
首先，确认是OSR模式下吗，如果是，界面是否加载正常。其次，89版本是否接口不一致

**Reply by bluecodeboy1987** [reply]
试过了，发现使用QCoreApplication方式是可以采用OSR模式的，改用QApplication GUI方式就不行

**Reply by bluecodeboy1987** [reply]
会一直报 GPU process isn't usable

**Reply by L-Super** [reply]
CMake配置是否正常？example能正常运行吗，正常来说，使用的是QApplication

**Reply by bluecodeboy1987** [reply]
正常的，CEF 提供的example运行正常的

**Reply by L-Super** [reply]
CEF cefclient正常，QCefView QCefViewTest不正常？

**Reply by bluecodeboy1987** [reply]
cefclient不正常，缺少gtk。我用的是cefsimple改成OSR模式

**Reply by L-Super** [reply]
QCefViewTest呢？是否为true？
```cpp
// main.cpp
config.setWindowlessRenderingEnabled(true);
```

**Reply by bluecodeboy1987** [reply]
QCefConfig config;
    config.setLogLevel(QCefConfig::LOGSEVERITY_DEFAULT);
    config.setBridgeObjectName("CallBridge");
    config.setRemoteDebuggingPort(9000);
    config.setBackgroundColor(Qt::lightGray);
    config.setWindowlessRenderingEnabled(true);

     config.addCommandLineSwitch("use-mock-keychain");
     config.addCommandLineSwitch("no-proxy-server");
    config.addCommandLineSwitchWithValue("renderer-process-limit", "1");
    QCefContext cefContext(&a, argc, argv, &config);
这是我的配置

**Reply by L-Super** [reply]
尝试使用Release模式试试

**Reply by bluecodeboy1987** [reply]
现在就是Release版，因为我Qt版本比较低5.6的所以测试用例没编译过，我适配了下信号槽问题

**Reply by L-Super** [reply]
版本低是有什么编译问题

**Comment by Code-Chen** [comment]
我在windows下遇到了同样的问题 请问最后你是如何解决的
